sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -22,6 +22,7 @@ internal object RadarSettings {
     private const val KEY_AD_ID_ENABLED = "ad_id_enabled"
     private const val KEY_TRACKING = "background_tracking"
     private const val KEY_TRACKING_OPTIONS = "tracking_options"
+    private const val KEY_PREVIOUS_TRACKING_OPTIONS = "previous_tracking_options"
```

Comment:
@corypisano How does previous tracking options play with remote tracking options?

Reply:
Same question.

Reply:
cc @bschoeneweis @corypisano @palmakkar

Reply:
_I think_ RTO will always keep priority (except a few blips).  If you have RTO enabled, it may get interrupted momentarily when you create the trip and these new options are set, but the following `/track` request would return the RTOs (assuming we always take the RTOs over the current tracking options).  This would repeat when completing the trip.  @corypisano may be able to add more details